Bottom: ae34c573eea6c9bd4a45ef136763597a2929f1ee
Top:    5870f04c6b7d8f1be21a691c838cd7ac0a1b763b
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-01-17 13:08:32 +0000

Refresh of drc-hack-hack-hack

---

diff --git a/src/find.c b/src/find.c
index 31034e5..bd8c934 100644
--- a/src/find.c
+++ b/src/find.c
@@ -2507,7 +2507,7 @@ IsPolygonInPolygon (PolygonType *P1, PolygonType *P2)
               line.Thickness = Bloat;
               /* Another Bloat is added by IsLineInPolygon, making the correct
                * 2x Bloat. Ideally we would change it there, but doing so
-               * breaks some other DRC checks which rely on the broken behaviour
+               * breaks some other DRC checks which rely on the  behaviour
                * in IsLineInPolygon.
                */
               line.Clearance = 0;
